components/automate-ui/src/app/pages/api-token/details/api-token-details.component.ts

```diff
@@ -104,6 +104,18 @@ export class ApiTokenDetailsComponent implements OnInit, OnDestroy {
           });
       }))
       .subscribe();
+
+    this.store.pipe(
+      select(updateStatus),
+      takeUntil(this.isDestroyed),
+      filter(state => this.saveInProgress && !pending(state)))
+      .subscribe((state) => {
+        this.saveInProgress = false;
+        this.saveSuccessful = (state === EntityStatus.loadingSuccess);
```

Comment:
[nit] Could use https://github.com/chef/automate/blob/master/components/automate-ui/src/app/entities/entities.ts#L45-L47 or add a singular `loadedSuccess` for consistency but this is fine.

Reply:
Haha! I think I made a similar comment on your PR. Here, it was just a singleton not an array so I chose not to use it, but I am also fine with either way.